js/scorm/wrapper.js

```diff
@@ -492,6 +492,13 @@ define([
         return;
       }
 
+      if ('value' in error.data) {
+        // because some browsers (e.g. Firefox) don't like displaying very long strings in the window.confirm dialog
+        if (error.data.value.length && error.data.value.length > 80) error.data.value = error.data.value.slice(0, 80) + '...';
```

Comment:
Just to point out the magic number of `80` and obviously doesn't take into account if someone's adjusted the error message. Probably best solved by switching to Notify and passing `value` into a read-only, scrollable text area.

Reply:
> Just to point out the magic number of `80` and obviously doesn't take into account if someone's adjusted the error message. Probably best solved by switching to Notify and passing `value` into a read-only, scrollable text area.
> 
> See [something similar](https://github.com/adaptlearning/adapt_authoring/blob/910073836a0278488e8e384fa5a88cfbe1a6debe/routes/lang/en.json#L301) in the authoring tool.

Yep, I'm not entirely happy with this but feel it's 'good enough' for the moment. Definitely like the idea of using Notify instead, let's get this in then make that change.